src/scripts/website-blocking/WebsiteBlocking.ts

```diff
@@ -3,7 +3,11 @@ import browser from 'webextension-polyfill';
 import WebsiteBlockingApi from "./WebsiteBlockingApi";
 import WebsiteBlockingConverter from "./WebsiteBlockingConverter";
 
-const db = browser.extension.getBackgroundPage().db;
+const db = () => browser.extension.getBackgroundPage().db;
+
+function youShallNotPass(tabId) {
```

Comment:
let's move this to yet another static function
this class is going to be shrunk down anyway